pmd-core/src/main/java/net/sourceforge/pmd/util/document/DocumentOperationsApplierForNonOverlappingRegions.java

```diff
@@ -85,7 +85,7 @@ public int compare(final DocumentOperation o1, final DocumentOperation o2) {
             return comparison;
         }
 
-        private boolean areInsertOperations(final RegionByLine r1, final RegionByLine r2) {
+        private boolean areInsertOperationsAtTheSameOffset(final RegionByLine r1, final RegionByLine r2) {
```

Comment:
we keep not knowing (from these classes perspective) if these are insert operations. You either check the operation type, or you drop that from the name.